src/main/scala/com/sksamuel/scapegoat/Feedback.scala

```diff
@@ -7,7 +7,9 @@ import scala.tools.nsc.reporters.Reporter
 /** @author Stephen Samuel */
 class Feedback(consoleOutput: Boolean, reporter: Reporter, sourcePrefix: String) {
 
-  val warnings = new ListBuffer[Warning]
+  private val warningsBuffer = new ListBuffer[Warning]
+
+  def warnings: Seq[Warning] = warningsBuffer.toSeq
```

Comment:
In Scala 2.13, `ListBuffer` is no longer a `Seq`. First, I added `.toSeq` anywhere `warnings` was expected to be a Seq, but I think exposing a mutual attribute is not a good idea anyways.